[TW-790] Add  topic on what happens when resource limits are reached (#4473)
postmanlabs/postman-docs: src/pages/docs/administration/resource-usage.md

---
title: "About resource usage"
updated: 2022-12-19
contextual_links:
  - type: section
    name: "Additional Resources"
  - type: link
    name: "Postman Plans and Pricing"
    url: "https://www.postman.com/pricing/"
  - type: link
    name: "Resource Usage dashboard"
    url: "https://go.postman.co/billing/add-ons/overview"
---

> Postman provides you with a limited number of resources you can use each month, depending on your Postman plan. Monthly resources include calls to the Postman API, requests made by monitors and scheduled collection runs, mock server requests, and Cloud Agent requests. Your Postman plan also has other limits, such as the number of integrations you can create, the number of custom domains, and storage for uploaded images.

Reviewer: The first "limited" sounds a bit... unenthusiastic - like Postman is a limited tool. Maybe "given" or "specified"?

Author: It's hard to make limits sound fun 😄 I'll go with "specified"

To find out what the resource limits are for your plan, go to the [Postman Plans and Pricing page](https://www.postman.com/pricing/). To view the resources you are using, go to your [Resource Usage dashboard](https://go.postman.co/billing/add-ons/overview). You can view how close you are to your limits and when your monthly limits will reset. If you need more resources, you can [purchase an add-on](/docs/administration/billing/#purchasing-add-ons) or [upgrade your plan](/docs/administration/billing/#team-and-plan-changes).

Refer to the following sections to understand what happens when your resource usage reaches the limits set by your Postman plan.

## Contents

* [Mock usage](#mock-usage)
* [Monitoring usage](#monitoring-usage)
* [Custom domains](#custom-domains)
* [Storage usage](#storage-usage)
* [Postman API usage](#postman-api-usage)
* [Integrations](#integrations)
* [Cloud Agent usage](#cloud-agent-usage)

> ## Mock usage

Author: I was following the UI here, but I think you're right it's better to follow the terminology we us in the LC.

> Learn more about [viewing and managing your monitor usage](/docs/monitoring-your-api/monitor-usage/).

## Custom domains

Postman Basic, Professional, and Enterprise plans give you a limited number of [custom domains](/docs/publishing-your-api/custom-doc-domains/) you can use when publishing API documentation. After the limit is reached, you won't be able to add a new custom domain.

![Custom domains limit reached](https://assets.postman.com/postman-docs/v10/usage-error-custom-domains-v10.jpg)

To add a new custom domain, delete one of your existing custom domains, [purchase an add-on](/docs/administration/billing/#purchasing-add-ons), or [upgrade your plan](/docs/administration/billing/#team-and-plan-changes).

## Storage usage

> Your Postman plan gives you a limited amount of storage for [uploaded images](/docs/publishing-your-api/authoring-your-documentation/#uploading-an-image) in your API documentation. You will get an error message when attempting to upload an image that would exceed your storage limit.

Reviewer 2: ```suggestion
Your Postman plan gives you a limited amount of storage for [uploaded images](/docs/publishing-your-api/authoring-your-documentation/#uploading-an-image) in your API documentation. You will get an error message if you try to upload an image that would exceed your storage limit.
```

![Postman API usage error](https://assets.postman.com/postman-docs/v10/usage-error-api-v10.jpg)

To make more requests to the Postman API before your monthly limit resets, you can [purchase an add-on](/docs/administration/billing/#purchasing-add-ons) or [upgrade your plan](/docs/administration/billing/#team-and-plan-changes).

## Integrations

Your Postman plan gives you a limited number of [integrations](/docs/integrations/intro-integrations/) for connecting Postman to third-party tools you use for API development. After the limit is reached, you won't be able to add a new integration. You will get a notification in the Postman app when attempting to add a new integration.

<img alt="Integrations limit reached" src="https://assets.postman.com/postman-docs/v10/usage-error-integrations-v10.jpg" width="361px"/>

To add a new integration, delete one of your existing integrations or [upgrade your plan](/docs/administration/billing/#team-and-plan-changes).

> ## Cloud Agent usage

Reviewer 3: What do you think about adding your note "`> To learn more about the resources included with your Postman plan and what happens when you reach your usage limits, go to [About resource usage](/docs/administration/resource-usage/).`" to the Postman Cloud Agent section at `/docs/getting-started/about-postman-agent/#the-postman-cloud-agent`?

Author: Great idea! Done! 😀
